model/app.py

```diff
@@ -627,13 +669,17 @@ def solve(self):
         return solution
 
     def model_json(self):
-        return model_to_dict(self.model)
+        if self.wild_type_model is not None:
+            return jsonpatch.make_patch(model_to_dict(self.wild_type_model), model_to_dict(self.model)).patch
```

Comment:
the only concern I have is if serializing two models on every call would be slow, I remember having the serialization of models as a bottleneck in the past, but I'm not sure
did you make any benchmarking on that?

Reply:
I just finished with that. I was able to shave off 4 seconds, which is already an improvement I think